staging/src/k8s.io/client-go/testing/actions.go

```diff
@@ -356,9 +379,10 @@ type ListRestrictions struct {
 	Fields fields.Selector
 }
 type WatchRestrictions struct {
-	Labels          labels.Selector
-	Fields          fields.Selector
-	ResourceVersion string
+	Labels                labels.Selector
+	Fields                fields.Selector
+	ResourceVersion       string
+	RequiresInitialEvents bool
```

Comment:
Wasn't sure if this belonged in WatchRestrictions or the root WatchAction struct